src/index.js

```diff
@@ -107,7 +107,7 @@ const SnackBar: ISnackBar = {
     warnDeprecation(options, 'color', 'textColor');
 
     const text = options.text || options.title;
-    const { numberOfLines } = options;
+    const { numberOfLines = 2 } = options;
```

Comment:
Is this necessary? It seems to default in the native code instead, perhaps we can leave this undefined.

Reply:
you are right, it's unnecessary, handled on [this line](https://github.com/cooperka/react-native-snackbar/blob/b7f4d9afc9d6cad25a5bd4d542b9fa5a041daa98/android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java#L105) and [this line](https://github.com/cooperka/react-native-snackbar/blob/b7f4d9afc9d6cad25a5bd4d542b9fa5a041daa98/ios/RNSnackBarView.m#L254)